swagger/scylla_v2.json

```diff
@@ -4956,6 +4956,33 @@
         }
       }
     },
+    "/config/uuid_sstable_identifiers_enabled": {
+      "get": {
+        "description": "Return true if node is using uuid sstable format: me-3g7k_098r_4wtqo2asamoc1i8h9n-big-CompressionInfo.db",
+        "operationId": "find_config_uuid_sstable_identifiers_enabled",
```

Comment:
Idea: maybe we should add gh action step to run:
- `cd swagger`
- `make clean`
- `make`
- check if anything has changed (using git)

Reply:
@karol-kokoszka

Reply:
> Off-topic: regenerating all swagger files changes comments styling (example below) in about 1200, so we should perhaps regenerate all of them.

It doesn't if you use correct version of swagger.
It's here https://github.com/scylladb/scylla-manager/blob/88fd44597af881079d47970d215f64dbf967e3f7/install-dependencies.sh#L45

> Idea: maybe we should add gh action step to run:

> cd swagger
> make clean
> make
> check if anything has changed (using git)

Why not. Good idea.

Reply:
But in the commit `feat(scyllaclient): add api for config uuid_sstable_identifiers_enabled` we can still see that `gen/agent/models/node_info` was modified while `agent.json` wasn't. After that it is indeed regenerated in a commit which `agent.json` is modified.

Reply:
Also, looks like import:
```go
"github.com/scylladb/scylla-manager/swagger/gen/scylla-manager/client/operations"
```
is broken in `scylla_manager_clinet.go` (missing previously present v3 in the path).

Reply:
> But in the commit `feat(scyllaclient): add api for config uuid_sstable_identifiers_enabled` we can still see that `gen/agent/models/node_info` was modified while `agent.json` wasn't. After that it is indeed regenerated in a commit which modifies `agent.json`.

Thanks for pointing out, it was in different commit, mixed with other schema changes, moved it to the commit it belongs to.



> Also, looks like import:
> 
> ```go
> "github.com/scylladb/scylla-manager/swagger/gen/scylla-manager/client/operations"
> ```
> 
> is broken in `scylla_manager_clinet.go` (missing previously present v3 in the path).

That is weird I have checked it multiple times, probably broke it during rebase, file regenerated, looks good now.

Reply:
But still we can see that commit `fix(swagger): fix files generation` modifies `gen/agent/models/node_info` without modifying `agent.json`. On the other hand, commit `feat(scyllaclient): add api for config uuid_sstable_identifiers_enabled` generates `gen/agent/models/node_info` in a wrong way.  

It also seems like generating swagger files doesn't work when checked out to `feat(scyllaclient): add api for config uuid_sstable_identifiers_enabled` generates `gen/agent/models/node_info`.  Shouldn't commit `fix(swagger): fix files generation` come first?

Reply:
True, there was mess with commit, fixed that

Reply:
There are still 2 commits modifying `gen/agent/models/node_info`, and only 1 changing `agent.json`.
Let's do it in that order:
- commit fixing current problems with swagger definitions and regenerating related swagger files
- commit extending definitions with API for checking node's uuid format and regenerating related swagger files
- the rest of implementation

Let's keep changes in swagger definitions and regenerating swagger files in one commit. This approach will allow for introducing additional checks on gh actions described [here](https://github.com/scylladb/scylla-manager/pull/3461#discussion_r1272167585), but instead of doing it only on branch HEAD, we could do it for every new commit introduce in given PR.

Reply:
This could also help with catching things like commit `feat(scyllaclient): add api for config uuid_sstable_identifiers_enabled` does not compile.